src/Metable.php

```diff
@@ -41,6 +41,32 @@ public static function bootMetable()
         });
     }
 
+    /**
+     * Override: Get attribute from model, then meta
+     *
+     * @param string $key
+     * @return mixed
+     */
+    public function getAttribute($key)
+    {
+        return parent::getAttribute($key) ?? $this->getMeta($key);
```

Comment:
I strongly disagree with this change. 

- This "magic" shortcut behaviour is confusing. Magic methods should be predictable in their usage, in this case you are suddenly reading from multiple different sources. If you have code that is using this to read from a metadata key and at some point a database column by the same name is added to the table, suddenly that code will not behave the same way.
- It is also confusing in that you can read existing metadata from an arbitrary property, but you cannot write it. Doing so will likely cause unknown database column errors when trying to save the model). I likewise would not want a separate setAttribute override because that would mean any mistyped attribute is suddenly start silently creating metadata, instead of leading to errors.
- This greatly increases the chance of conflict with other libraries and existing code which also override this method, which at the very least makes this is a breaking change requiring a major version.
- This is it well outside the scope of what your PR is claiming to do: this has nothing to do with defaults.